integration-tests/helpers/embrace_server.ts

```diff
@@ -1,44 +1,50 @@
-import {start_mockserver, stop_mockserver} from "mockserver-node";
-import {EmbraceRequestBody, SessionPayload} from "../typings/embrace";
+import {EmbraceData} from "../typings/embrace";
+import axios from "axios";
+import {RAW_EMB_URL, SERVICE_NAME} from "./env";
 
-import {mockServerClient} from "mockserver-client";
+const EMB_URL = RAW_EMB_URL.replace("{servicename}", SERVICE_NAME);
 
-const PORT = 8877;
-
-const startServer = (trace: boolean) => {
-  start_mockserver({
-    serverPort: PORT,
-    trace,
-  });
-};
-
-const stopServer = () => {
-  stop_mockserver({
-    serverPort: PORT,
-  });
-};
-
-const clearServer = async () => {
-  return await mockServerClient("localhost", PORT).clear({}, "LOG");
-};
-
-const getSessionPayloads = async (delay = 2000): Promise<SessionPayload[]> => {
+const getSessionPayloads = async (
+  sessionId: string | string[],
```

Comment:
instead of the caller needing to figure out the session ID I think simpler if in `beforeTest` a DELETE request was made to the namespace to clear out any data generated by previous testing, then we could grab all session payloads here

Reply:
If you do that, you won't be able to run multiple instances using the same test APK. You will need to create different APKs with different embrace config

Reply:
hmm that's true, ok let's keep for now we may want to revisit when we hook things up to CI depending on how much we run concurrently